### DISCUSSION TITLE
SoftEther VPN server Local Bridge on FreeBSD

### DISCUSSION THREAD

**Discussion by eddietorial** [SoftEtherVPN/SoftEtherVPN#1671] [Q&A] [answered]
I'm using SoftEther VPN Server Developer Edition v5.02 Build 5180 on FreeBSD 13.1-RELEASE-p2. I have attempted to to create a local bridge via the Windows server manager GUI. However, I cannot tell whether it is operational or not.

The bridge does not show up on FreeBSD. Is there some additional manual configuration required?

@metalefty, would you mind sharing how you got it to work? 
https://github.com/SoftEtherVPN/SoftEtherVPN/issues/1046#issuecomment-561495542

**Comment by metalefty** [comment]
What's your network card and interface name (e.g. re0)?

**Comment by eddietorial** [comment, marked as answer]
The FreeBSD server is on AWS. The network interface name is `xn0`.

**Reply by metalefty** [reply]
I confirmed local bridges can be configured but local bridge doesn't work at all on the cloud even if it is configured.

Local bridge requires promiscuous mode however promiscuous mode is prohibited on EC2. I think you should use SecureNAT.

![image](https://user-images.githubusercontent.com/941609/194281750-c56f8f44-a425-4d77-8c66-ad187e7efdbc.png)

**Reply by eddietorial** [reply]
Interesting! Good to know. Thanks for checking. I'll use SecureNAT.
